cli/client_test.go

```diff
@@ -74,3 +84,70 @@ func TestListOrganizationsAction(t *testing.T) {
 	test.That(t, out.messages[1], test.ShouldContainSubstring, "jedi")
 	test.That(t, out.messages[2], test.ShouldContainSubstring, "mandalorians")
 }
+
+func TestTabularDataByFilterAction(t *testing.T) {
+	pbStruct, err := protoutils.StructToStructPb(map[string]interface{}{"bool": true, "string": "true", "float": float64(1)})
+	test.That(t, err, test.ShouldBeNil)
+
+	// calls to `TabularDataByFilter` will repeat so long as data continue to be returned,
+	// so we need a way of telling our injected method when data has already been sent so we
+	// can send an empty response
+	dataRequested := false
```

Comment:
[super-nit] zero value 
```suggestion
	var dataRequested bool
```